lints/lint_dnsname_contains_question_marks.go

```diff
@@ -0,0 +1,48 @@
+// lint_dnsname_contains_question_marks.go
```

Comment:
Delete this line, add a copyright notice.

Reply:
Copyright?